stable/konfigurator/values.yaml

```diff
@@ -0,0 +1,33 @@
+konfigurator:
```

Comment:
If this is the only component, move the values to the root level

Reply:
Updated